### ISSUE TITLE
exit codes for for grafana-server status

### ISSUE THREAD

**Issue by yaryarrr** [grafana/grafana#2106]
I'm attempting to deploy grafana via configuration management (in my case salt, but applies to all) and it won't start the service because the init script packaged in the RPM doesn't provided correct status'.

Here are the return codes used: http://refspecs.linuxbase.org/LSB_3.1.0/LSB-Core-generic/LSB-Core-generic/iniscrptact.html

**Example:**
```
[root@localhost ~]# service grafana-server status
grafana-server is stopped
[root@localhost ~]# echo $?
0
[root@localhost ~]# service grafana-server start
Starting Grafana Server: .... nohup: redirecting stderr to stdout
OK
[root@localhost ~]# service grafana-server status
grafana-server (pid  3513) is running...
[root@localhost ~]# echo $?
0
```

**Comment by torkelo** [comment]
what exist codes do you expect? the link does say

**Comment by yaryarrr** [comment]
Ack. sorry...I sent the wrong link (too many things at once). I updated the original link, does that help?

**Comment by damm** [comment]
```
0	program is running or service is OK
1	program is dead and /var/run pid file exists
2	program is dead and /var/lock lock file exists
3	program is not running
4	program or service status is unknown
```

**Comment by torkelo** [comment]
Here are the init.d scripts: 
(debian)
https://github.com/grafana/grafana/blob/master/packaging/deb/init.d/grafana-server

(centos/rpm)
https://github.com/grafana/grafana/blob/master/packaging/rpm/init.d/grafana-server

PR please?

**Comment by torkelo** [comment]
fixed by #2115
